presto-iceberg/pom.xml

```diff
@@ -200,14 +200,9 @@
 
         <!-- for testing -->
```

Comment:
this was previosly by testng

Reply:
I guess this comment concerns the whole section, not a single dependency.